### PR TITLE
Vagrant Command Prompt Opens Without Permission

### PR DESCRIPTION
**Pull request by Rlisa33**
Everytime the computer turns on Vagrant command prompt opens. This can be cause by various reasons. An open issue discusses how this is resolved.
image

### CODE REVIEW

File: pages/Rlisa33/Issue1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+###The Issue of Vagrant Command Prompt Opening as Soon as Windows Screen Is On!
```
Comment (mappuji) [inline pages/Rlisa33/Issue1]: It is not an issue, it is how our program works based on our script.

File: pages/Rlisa33/Issue1 [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+This is very important and can occur because of various reasons.
+*The computer software for vagrant is not installed properly.
```
Comment (mappuji) [inline pages/Rlisa33/Issue1]: I don't think this is a problem.

File: pages/Rlisa33/Issue1 [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+*The computer has not accepted or gave permission to your software download
+* The computer needs to download software required to install vagrant.
+_A Step By Step Process Explains the Confusion with unexplained Opening of Command Prompt when the Windows screen is up!_
```
Comment (mappuji) [inline pages/Rlisa33/Issue1]: Could you explain more about this one?

**Comment by kylemathias** [comment]
That console opening up is a script you setup during the First Steps. This is one of the only ways to get Vagrant to auto start. I am not sure what the problem is that you are trying to convey.

**Comment by paulbert** [comment]
Hi @Rlisa33 could you provide a screenshot of the command prompt error?  I think adding some of this information might be helpful for a "troubleshooting" section at the bottom of the [Windows install page](http://open-learning-exchange.github.io/#!pages/vagrantwin.md), but a screenshot would help us understand the error that you're experiencing so we can help guide what to add to the documentation.

**Comment by Rlisa33** [comment]
image

**Comment by sarvanivadali** [comment]
Hi @Rlisa33 This pull request seens inactive. Closing it now. Please open it when you come back.

**Pull request closed without merging**